lib/src/components/NxLoadingSpinner/__tests__/NxLoadingSpinner.test.tsx

```diff
@@ -5,22 +5,26 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { shallow } from 'enzyme';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 import NxLoadingSpinner from '../NxLoadingSpinner';
 
 describe('NxLoadingSpinner', function() {
+  const quickRender = rtlRender(NxLoadingSpinner, {}),
+      renderEl = rtlRenderElement(NxLoadingSpinner, {});
+
   it('renders an nx-loading-spinner__icon within a nx-loading-spinner', function() {
```

Comment:
Rather than the classname, just query that it renders something with a role of `status`

Reply:
> Rather than the classname, just query that it renders something with a role of `status`

Added that here: 0c62a49fa177840bdb1706b2154d66bd593106ab
Should I still keep the `querySelector` for `.nx-loading-spinner__icon`?

Reply:
> Should I still keep the querySelector for .nx-loading-spinner__icon?

No

Reply:
> > Should I still keep the querySelector for .nx-loading-spinner__icon?
> 
> No

Removed: 947237f2ee4be40b42e3565847d35d3840a47452